src/CodeGenerator/Program.cs

```diff
@@ -24,17 +24,55 @@ static void Main(string[] args)
             {
                 outputPath = AppContext.BaseDirectory;
             }
+
+            string libraryName;
+            if (args.Length > 1)
+            {
+                libraryName = args[1];
+            }
+            else
+            {
+                libraryName = "cimgui";
+            }
+
+            string projectNamespace = libraryName switch
+            {
+                "cimgui" => "ImGuiNET",
+                "cimplot" => "ImGuiNET",
+                "cimnodes" => "ImGuiNET",
+                "cimguizmo" => "ImGuiNET",
+                _ => throw new NotImplementedException()
+            };
+
+            string classPrefix = libraryName switch
+            {
+                "cimgui" => "ImGui",
+                "cimplot" => "ImPlot",
+                "cimnodes" => "ImNodes",
```

Comment:
After looking into this more, `cimnodes` actually wraps the [imnodes](https://github.com/Nelarius/imnodes) project, which is similar but different to another project called [ImNodes](https://github.com/rokups/ImNodes). Using `ImNodes` as the class name (instea of imnodes) might be confusing because of that.